src/GitHub/Data/PullRequests.hs

```diff
@@ -180,8 +184,22 @@ instance Binary PullRequestReference
 -- JSON instances
 -------------------------------------------------------------------------------
 
+-- | Helper function, reads either the "assignee" OR "assigneed" OR
+-- both from a JSON object.
+getAssignees :: Object -> Parser (Vector SimpleUser)
```

Comment:
Is there a downside to including both? If we include both then it will work either way. You yourself earlier posted an example which contained both keys. I'm fine removing it but this seems like the way to accommodate all configurations which are likely to appear.